script/collect.js

```diff
@@ -118,11 +118,14 @@ async function main () {
   const oldNpmCount = old.find(release => release.npm_package_name === 'electron').length
   const newNpmCount = releases.find(release => release.npm_package_name === 'electron').length
 
+  const diffNotesCount = old.filter((release, index) => release.body !== releases[index].body).length
```

Comment:
You could improve the performance of this by using `.some` instead of `.filter` and length!